README.md

```diff
@@ -103,3 +103,39 @@ You don't need to push the mattermost-operator image to DockerHub or any other r
 ```bash
 $ kind load docker-image mattermost/mattermost-operator:test
```

Comment:
Doesn't this need to be done after the new instructions? If not, does this line need to be removed?